x-pack/plugins/cloud_security_posture/public/pages/configurations/findings_flyout/findings_flyout.tsx

```diff
@@ -212,7 +246,12 @@ export const FindingsRuleFlyout = ({
         </EuiTabs>
       </EuiFlyoutHeader>
       <EuiFlyoutBody key={tab.id}>
-        <FindingsTab tab={tab} findings={findings} />
+        {['overview', 'rule'].includes(tab.id) && (
+          <div style={{ marginBottom: 16, borderRadius: 4, overflow: 'hidden' }}>
```

Comment:
should it be a part of `MissingFieldsCallout` if we don't reuse it anywhere?

Reply:
I prefer to keep the CSS for placement outside of the component, so the margin should be managed in the wrapper. However, the border radius is considered part of the component and should be applied within it. Thanks for noticing that